### PR TITLE
Env variable to test fixed grid size with SK kernels

### PR DESCRIPTION
**Pull request by AlexBrownAMD**
Small change to add another environment variable to make testing and benchmarking stream-k performance easier.

### CODE REVIEW

File: Tensile/Source/lib/source/ContractionSolution.cpp [hunk 1]
```diff
@@ -1604,6 +1604,8 @@ namespace Tensile
     size_t ContractionSolution::getSKGrid(Hardware const& hardware, size_t tiles) const
     {
         AMDGPU const* pAMDGPU = dynamic_cast<AMDGPU const*>(&hardware);
+        if(pAMDGPU->skFixedGrid > 0)
```
Comment (nakajee) [inline Tensile/Source/lib/source/ContractionSolution.cpp:1607]: Don't we need max value check below (skMaxCUs or tiles)?
In other words, can we use any skFixedGrid value without limitation?

Comment (AlexBrownAMD) [reply Tensile/Source/lib/source/ContractionSolution.cpp:1607]: Yea this overrides all other options so we can test grids both larger and smaller than CU count. This option makes it easier to script benchmarking sweeps to analyze performance at different grid sizes.

Comment (nakajee) [reply Tensile/Source/lib/source/ContractionSolution.cpp:1607]: OK. Thanks for confirmation.

**Review by nakajee — APPROVED** [review]
I do not have further comments.
Not sure why extended test is not triggered, but most likely this change will not affect extended test result.

**Comment by TorreZuk** [comment]
> I do not have further comments. Not sure why extended test is not triggered, but most likely this change will not affect extended test result.

I think everyone has to add ci:extended label now if you want extended pipelines to run

**Comment by nakajee** [comment]
> > I do not have further comments. Not sure why extended test is not triggered, but most likely this change will not affect extended test result.
> 
> I think everyone has to add ci:extended label now if you want extended pipelines to run

OK. Thanks for letting me know. I was not aware of that. 
I will keep that in mind.
For this PR, I think we can skip extended test.

**Pull request merged**